policyengine_uk/variables/household/demographic/benunit/benunit_count_children.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_uk.model_api import *
+
+
+class benunit_count_children(Variable):
+    value_type = int
+    entity = BenUnit
+    label = "Number of children in the Benefit Unit"
```

Comment:
Could we decapitalise all labels too? (except for those starting with a proper noun like Housing Benefit)

Reply:
Decapitalise the first word as well?
E.g.
```
"Number of children in the benefit unit"
```
or
```
"number of children in the benefit unit"
```

Reply:
Can't speak for Nikhil, but I believe so, as I believe he prefers lowercase in the app variable tree

Reply:
As a general rule of thumb I think you should be able to drop `label` as-is mid-sentence in 'X entity's VARIABLE_LABEL is Y'

Reply:
That makes sense, I was going off of the idea that when we search for variables on the household side the labels just appear as VARIABLE_LABEL but I am fairly indifferent so I will change to lowercase